libs/community/langchain_community/document_loaders/sharepoint.py

```diff
@@ -33,10 +33,6 @@ class SharePointLoader(O365BaseLoader, BaseLoader):
     """ Whether to load authorization identities."""
     token_path: Path = Path.home() / ".credentials" / "o365_token.txt"
     """ The path to the token to make api calls"""
-    file_id: Optional[str] = None
-    """ The ID of the file for which we need auth identities"""
-    site_id: Optional[str] = None
-    """ The ID of the Sharepoint site of the user where the file is present """
```

Comment:
NOTE: Our team added the two fields above, but they are no longer used anywhere in the file.